beacon-chain/rpc/apimiddleware/endpoint_factory.go

```diff
@@ -37,6 +37,7 @@ func (_ *BeaconEndpointFactory) Paths() []string {
 		"/eth/v1/beacon/pool/proposer_slashings",
 		"/eth/v1/beacon/pool/voluntary_exits",
 		"/eth/v1/beacon/pool/sync_committees",
+		"/eth/v1/beacon/weak_subjectivity",
```

Comment:
Similar to my comment of the proto service, can we move this endpoint out of the `eth` namespace? It can still reside in this list of endpoints.